Merge throws error on successful merge

Hi,

I have this workflow:

```yml
name: Test Pipeline

on: [pull_request]

jobs:
  build-and-test:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
      with:
        repository: 'ryukote/carbooking-api' 
        fetch-depth: '0' 
    - name: Setup .NET Core
      uses: actions/setup-dotnet@v1
      with:
        dotnet-version: 3.1.101
    - name: Build with dotnet
      run: dotnet build ./ConsoleApp1/ConsoleApp1.sln --configuration Release
    - name: Test with dotnet
      run: dotnet test ./ConsoleApp1/ConsoleApp1.sln
      
  pull-request:
    needs: build-and-test
    runs-on: ubuntu-latest
    steps:
    - name: Merge Pull Request
      uses: sudo-bot/action-pull-request-merge@v1.0.3
      with:
        github-token: "${{ secrets.TOKEN }}"
        number: ${{ github.event.pull_request.number }}  
        merge-method: merge
        merge-title: "Merge #${{ github.event.pull_request.number }}"
        merge-message: "Merge #${{ github.event.pull_request.number }}"

```

Your action merges pull request, but for some reason it throws error "##[error]Label does not exist".
Cause of that it says workflow didn't pass, even tho pull request was merged.

Do you maybe have any idea what should I do?

Thank you.

--- a/.github/workflows/merge.yml
+++ b/.github/workflows/merge.yml
@@ -8,7 +8,7 @@ jobs:
         runs-on: ubuntu-latest
         steps:
             - name: merge pull request
-              uses: sudo-bot/action-pull-request-merge@v1.0.3
+              uses: sudo-bot/action-pull-request-merge@v1.1.0
               with:
                   github-token: ${{ secrets.GITHUB_TOKEN }}
                   number: ${{ github.event.pull_request.number }}

